packages/eslint-plugin-change/rules/prefer-object-spread-to-lodash.js

```diff
@@ -1,17 +1,18 @@
 const isLodashCall = node => node.callee && node.callee.type === 'MemberExpression' && node.callee.object.name === '_';
-const isExtendOrAssignInCall = node =>
-  isLodashCall(node) && (node.callee.property.name === 'extend' || node.callee.property.name === 'assignIn');
+// eslint-disable-next-line prettier/prettier
```

Comment:
does prettier fight with eslint on the correct formatting of this line? If so, that might indicate that the config isn't quite right...

Reply:
Yeah, it does...